integreat_cms/cms/views/utils/hix.py

```diff
@@ -59,13 +60,21 @@ def get_hix_score(request, region_slug):
     # Don't pass texts larger than 100kb to the api in order to avoid being vulnerable to dos attacks
     if len(request.body) > MAX_TEXT_LENGTH:
         return JsonResponse({"error": "Request too large"})
+
     body = json.loads(request.body.decode("utf-8"))
     text = body["text"]
 
     if not isinstance(text, str) or not text.strip():
         logger.warning("Received invalid text: %r", text)
         return JsonResponse({"error": f"Invalid text: '{text}'"})
 
-    if score := lookup_hix_score(text):
+    # convert special characters into plain letters
+    unescaped = html.unescape(text.replace("\n", "\r\n"))
```

Comment:
> f "\n" appears in the text by chance, for example when an user writes "Please use \n as line break" in the editor, simple replacement might replace it too, even though it is not intended as a line break. I testet it and "\n" tipped in the editor was not replaced, but I cannot garant it...

This is totally fine, since a newline `\n` is one single special character, whereas when "\n" is typed in the content, it is considered two separate characters `\` and `n`, which is escaped in the content to `\\n` and not replaced with this statement :+1: